assets/styles/styles.scss

```diff
@@ -0,0 +1 @@
+@import "../js/link/style.scss"
```

Comment:
also, the choice of import has

1. [#using-css](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/#using-css), it generates the file name under build, file name `style-[fileName].css`
2. (CURRENT) @import "../js/link/style.scss" , it generates the file name under build, file name `style.scss.css` 

what's your preference?